src/collective/easyform/api.py

```diff
@@ -158,7 +158,11 @@ def set_fields(context, schema):
     snew_schema = serializeSchema(schema)
     # store the current schema
     context.fields_model = snew_schema
+    # Plain reindexObject() will call notifyModified and then reindex all.
+    # We know nothing has changed that needs indexing,
+    # except that we must update the modified date, and reindex its index.
     context.notifyModified()
+    context.reindexObject(idxs=["modified"])
```

Comment:
LGTM, but I didn't  know about .notifyModified() until now.  Is the issue that when you add the idxs parameter, reindexObject does not call notifyModified? And what is is exactly doing, triggering the event which cachepurging also might listen to?

Reply:
I will answer the question about notifyModified, but first I will rebase and force push.

Reply:
I wondered if the comments would disappear after a forced push, which they don't.
Let me answer the question in the main PR.